### ISSUE TITLE
Shyfox 3.8.1 dying after firefox develloper edition update

### ISSUE THREAD

**Issue by claymorwan** [Naezr/ShyFox#170]
I just updated firefox developer edition and it started doing this
I'm on Shyfox 3.8.1 with firefox developer edition v132
![image](https://github.com/user-attachments/assets/e4993528-faac-4547-989e-51f571d8d48f)
![Screenshot_20241001_182922](https://github.com/user-attachments/assets/a01d1e4f-7883-4f9f-9313-7b34557d1c67)

**Comment by Naezr** [comment]
fix uploaded

**Comment by claymorwan** [comment]
Yep it's fixed but now my sidebar doesn't wanna hide anymore

**Comment by Naezr** [comment]
Reset Userchrome Toggle settings

**Comment by claymorwan** [comment]
Just did and now that for some reason, I did set back the settings correctly

https://github.com/user-attachments/assets/3f8183b4-6295-450e-81bd-ce99812d9144

**Comment by Naezr** [comment]
fix uploaded

**Comment by claymorwan** [comment]
Seems to working now, thanks :3

**Comment by claymorwan** [comment]
Welp it's dead agin, the navbar and toolbar this time

**Comment by MatlabRes** [comment]
shyfox 3.8.1
mozilla 132
scroll bar fix to right side on full height ignore right panel and navigation 
back/forward/reset/home buttons lost background

**Comment by H1DDENADM1N** [comment]
> Welp it's dead agin, the navbar and toolbar this time, they work on normal but not on hide mode

Same Problem on Windows10 with Firefox Developwe Edition 133.0 x64.

![PixPin_2024-10-31_11-54-53](https://github.com/user-attachments/assets/b141196d-699a-48e3-ba09-668e57f68814)

**Comment by jaashuu** [comment]
yeah, i have a similar problem on Firefox Developer Edition where the toolbar is going behind the page content 133.0b1
